### PR TITLE
PWX-26400: Updating sample app to work without region for fb and upda…

### PR DESCRIPTION
**Pull request by pp511**
…ting cr yaml

Signed-off-by: Priyanshu Pandey <ppandey@purestorage.com>

What this PR does / why we need it:
PWX-26400 [https://portworx.atlassian.net/browse/PWX-26400]

Which issue(s) this PR fixes (optional)
Sample app doesn't work without region in the sample yaml for pure fb.
Fixing the yaml to use separate pba objects in the app.

Special notes for your reviewer:

Provided second copy of pba and pbc for pure fb.
Updated the sample app to pass "default" region in case of missing region as s3 APIs expect a non empty region field.
Provided a separate yaml for fb.
Fixed the bucket access in the sample yaml.

### CODE REVIEW

File: examples/sample-app/main.go [hunk 1]
```diff
@@ -30,7 +35,13 @@ func main() {
 		if err != nil {
 			fmt.Printf("bad credentials: %s", err)
```
Comment (trierra) [inline examples/sample-app/main.go:36]: use `logrus.Errorf()` here

File: examples/sample-app/main.go [hunk 2]
```diff
@@ -30,7 +35,13 @@ func main() {
...
+		cfg := aws.NewConfig().WithEndpoint(endpointStr).WithDisableSSL(true).WithCredentials(creds).WithS3ForcePathStyle(true)
+		if len(regionStr) > 0 {
+			cfg = cfg.WithRegion(regionStr)
+		} else {
+			cfg = cfg.WithRegion(DefaultRegion)
+		}
+
 		svc := s3.New(session.New(), cfg)
 		objName := uuid.New().String()
 		fmt.Printf("--- PUT OBJECT %s IN BUCKET %s ---\n", objName, bucketName)
```
Comment (trierra) [inline examples/sample-app/main.go:47]: same here `logrus.Infof()`

**Comment by trierra** [comment]
lgtm

File: examples/sample-app/main.go (hunk 1 above)
Comment (pp511) [reply examples/sample-app/main.go]: ack.

File: examples/sample-app/main.go (hunk 2 above)
Comment (pp511) [reply examples/sample-app/main.go:48]: I have introduced logrus in this package and updated the error logs above. However for this particular print Grant did write it in a particular format using fmt to make it look pretty I think. Also this is the sample app so it is fine I think. I will let Grant take the call if he wants to update this.

**Pull request merged**